specification/botservice/resource-manager/Microsoft.BotService/preview/2018-07-12/botservice.json

```diff
@@ -1012,6 +1012,283 @@
           "nextLinkName": "nextLink"
         }
       }
+    },
+    "/providers/Microsoft.BotService/checkECNameAvailability": {
```

Comment:
why not full name? There are people that consume the API and readability of the paths is quite valuable. I'd root for checkEnterpriseChannelNameAvailability

Reply:
Sounds good, changed.